test/translate.test.js

```diff
@@ -0,0 +1,37 @@
+var expect = require('chai').expect;
+var striptags = require('striptags');
+var translate = require('google-translate-api');
+
+describe('translate()', function() {
+
+    test('should translate a Finnish text to right English text', async (done) => {
+        let finnishText = 'Pienen vilauksen lisäksi videolla kuullaan hieman hahmon motivaatiota sodanlietsomiselle,';
+        let finnishTextEnglishTranslation = 'In addition to a small glimpse, the video is a little bit heard about the motivation of a figure for war-firing,';
+
+        // const translation = await translate('finnishText');
```

Comment:
I guess we don't need this comment?

Reply:
Yep, that's not needed. Forgotten when refactored.